transformer_engine/common/include/transformer_engine/multi_tensor.h

```diff
@@ -17,13 +17,58 @@
 extern "C" {
 #endif
 
+/*!  \brief Computes L2 norm for a list of contiguous tensors, providing the option to
+ *          immediately exit the kernel based on the value of the 'noop' tensor.
```

Comment:
We should make the brief description brief. It would be better to describe the noop logic when documenting the `noop_flag` arg.
```suggestion
/*!  \brief Compute L2 norm for a list of tensors
```